ecommerce/frontend/src/components/profile/ProfileDetails.less

```diff
@@ -0,0 +1,7 @@
+ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    text-align: center;
+}
```

Comment:
This will change all unordered lists in the page which is bad. If you are going to use `ul`s you need to make a className for that list.